README.md

```diff
@@ -19,22 +24,39 @@ for publication.  A preprint can be obtained from Doug Barrick (barrick@jhu.edu)
 [License](LICENSE.txt)
 
 ## Setup
-To run the code, follow the instructions below:
+To run the most recent code, follow the instructions below:
 1. git clone the repository:
 
     ```git clone https://github.com/barricklab-at-jhu/Ising_programs.git```
-1. Create and activate a new environment (requires conda or miniconda):
+  
+2. Create and activate a new environment (requires conda or miniconda):
+    ```bash
+    conda env create -f environment.yml
+    conda activate ising_py3
+    ```
+   
+To run a version of the code linked to the publication, <insert protein science link> follow these instructions:
```

Comment:
Here is the snippet linking readers to the publication branch. I'll create this after we merge into master